e2e/test/scenarios/sharing/alert/email-alert.cy.spec.js

```diff
@@ -64,25 +64,60 @@ describe("scenarios > alert > email_alert", { tags: "@external" }, () => {
       table: PEOPLE_ID,
     });
 
-    cy.icon("bell").click();
-
-    cy.findByRole("dialog").within(() => {
-      cy.findByLabelText("Name").type(" alert");
-      cy.findByRole("button", { name: "Save" }).click();
-    });
-
-    cy.wait("@saveCard");
+    saveAlert();
 
-    cy.findByTestId("alert-education-screen").within(() => {
-      cy.findByRole("button", { name: "Set up an alert" }).click();
+    cy.findByTestId("toast-undo").within(() => {
+      cy.findByText("Your alert is all set up.").should("be.visible");
     });
-    cy.findByRole("button", { name: "Done" }).click();
 
     cy.wait("@savedAlert").then(({ response: { body } }) => {
       expect(body.channels[0].channel_type).to.eq("email");
       expect(body.channels[0].enabled).to.eq(true);
     });
   });
+
+  it("should enable alert to be updated (without updating question) (metabase#36866)", () => {
+    openTable({
+      table: PEOPLE_ID,
+    });
+
+    saveAlert();
+
+    cy.log("Check that /api/card has been called once");
+    cy.get("@saveCard.all").should("have.length", 1);
+
+    cy.findByTestId("toast-undo").within(() => {
+      cy.findByText("Your alert is all set up.").should("be.visible");
+    });
+
+    clickAlertBell();
+
+    cy.findByTestId("popover").within(() => {
+      cy.findByText("You set up an alert").should("be.visible");
+      cy.findByText("Edit").click();
+    });
+
+    cy.log("Change the frequency of the alert to weekly");
+
+    cy.get(".Modal--full").within(() => {
+      cy.findByText("How often should we check for results?")
+        .parent()
+        .parent()
+        .within(() => {
+          const buttons = cy.findAllByTestId("select-button");
+          buttons.should("have.length", 2);
+          buttons.eq(0).click();
```

Comment:
Perhaps there's an opportunity for refactoring the alert modal, since I think `data-testid` is typically a unique identifier.

Reply:
We don't usually enforce uniqueness. There are a lot of cases where we have lists of items with the same test id.